tests/dispenser/TestLimitCommand.test_limit_command_success.False.True.approved.txt

```diff
@@ -0,0 +1,3 @@
+DEBUG: HTTP Request: GET https://snapshottest.dispenser.com/fund/0/limit "HTTP/1.1 200 OK"
+DEBUG: Converted response microAlgos to Algos: 1.0
+Remaining daily fund limit: 1.0
```

Comment:
Should we add `Algo` like we do now when printing the successfully funded message?